00 Meta/Vault Infos/Introduce Use Cases.md

```diff
@@ -0,0 +1,44 @@
+# Introduce Use Cases
+
+In order to make the vault better navigable and give some more context to available queries as well as inspiration how these queries could be used, we want to introduce **Use Cases**. 
+
+## What's a good topic for a Use Case?
+
+Like the name says, a Use Case should be about some specific scenario you could achieve with dataview which requires (or can require) more than one dataview query. They should be **overviews for a topic**. Some examples:
```

Comment:
[...] specific scenario you could achieve with dataview**,** which requires [...]